changelog.d/15301.misc

```diff
@@ -0,0 +1 @@
+Revert "check sqlite database file exists before porting" PR #14692.
```

Comment:
-> .bugfix, something like: "Fix a bug introduced in Synapse XYZ where the [SQLite port_db script](link to docs) would fail to open the SQLite database".